### DISCUSSION TITLE
Querybar gadgets

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#717] [Ideas]
**Clock**
![flow01](https://user-images.githubusercontent.com/6903107/135663615-e15535ab-5a00-4b64-8af5-0d844048325d.jpg)

I thought about it while taking a shower. (I thought I was a genius 😁) What do you think?
I can make this much come from some themes. If the response is good, How about we move on?

**Weather**
![flow02](https://user-images.githubusercontent.com/6903107/135663819-86e9e50d-724c-492f-840f-b14613beff67.jpg)

**Small Shortcut**
![flow03](https://user-images.githubusercontent.com/6903107/135663862-40c9db65-f5cf-4dd7-b4b0-44669e6165f1.jpg)

imagine, music player, stock ticker, etc...

We can make more area for gadgets like this. 
![image](https://user-images.githubusercontent.com/6903107/135663971-69657a59-7881-44a0-9f59-6279ee43fd55.png)

**Comment by taooceros** [comment]
I think that looks great! Though that may require quite a bit additional works 🤣

**Reply by taooceros** [reply]
If this is implemented, we may need to keep a wider search bar.

**Reply by onesounds** [reply]
I don't think so. it's gone when you start typing query. it just showing when no query status.
Maybe I should make a watch and show it to you.

**Reply by onesounds** [reply]
yeah, If we trying to do more than a clock,  You will want to give up.

**Reply by taooceros** [reply]
> I don't think so. it's gone when you start typing query. it just showing when no query status. Maybe I should make a watch and show it to you.

that's sounds reasonable.

**Comment by Zeroto521** [comment]
Or a simple way that we could directly replace the magnifier icon with a specific plugin icon.

![image](https://user-images.githubusercontent.com/25895405/135699870-3228c4c9-2442-49d5-b1b0-51031dee07cc.png)

**Reply by onesounds** [reply]
> Or a simple way that we could directly replace the magnifier icon with a specific plugin icon.

Right, I think both should be able to turn it off and on individually.

**Comment by jjw24** [comment]
I think it's great. my only concern is watch out for memory/cpu usage (i dont know if this addition will impact) and test it as part of the initial work
